### PR TITLE
Add Kubeflow 1.9 release roadmap

### PR DESCRIPTION
**Pull request by rimolive**

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -1,6 +1,26 @@
 # Kubeflow Roadmap
 
-## Kubeflow 1.8 Release, Planned for release: Oct 2023
+## Kubeflow 1.9 Release, Planned fro release: Jul 2024
+The Kubeflow Community plans to deliver its v1.8 release in Jul 2023 per this [timeline](https://github.com/kubeflow/community/blob/master/releases/release-1.9/README.md#timeline). The high level deliverables are tracked in the [v1.9 Release](https://github.com/orgs/kubeflow/projects/61) Github project board. The v1.9 release process will be managed by the v1.9 [release team](https://github.com/kubeflow/community/blob/master/releases/release-1.9/release-team.md) using the best practices in the [Release Handbook](https://github.com/kubeflow/community/blob/master/releases/handbook.md).
```
Comment (jstourac) [inline ROADMAP.md:4]:
```suggestion
The Kubeflow Community plans to deliver its v1.9 release in Jul 2024 per this [timeline](https://github.com/kubeflow/community/blob/master/releases/release-1.9/README.md#timeline). The high level deliverables are tracked in the [v1.9 Release](https://github.com/orgs/kubeflow/projects/61) Github project board. The v1.9 release process will be managed by the v1.9 [release team](https://github.com/kubeflow/community/blob/master/releases/release-1.9/release-team.md) using the best practices in the [Release Handbook](https://github.com/kubeflow/community/blob/master/releases/handbook.md).
```

**Comment by rimolive** [comment]
@jbottum We need approvals for this PR to get merged.

**Comment by jbottum** [comment]
@zijianjoy can you please review ?   Note - we need to remove and/or replace pdmack from approvers.

File: ROADMAP.md [hunk 2]
```diff
@@ -1,6 +1,26 @@
 # Kubeflow Roadmap
 
-## Kubeflow 1.8 Release, Planned for release: Oct 2023
+## Kubeflow 1.9 Release, Planned for release: Jul 2024
+The Kubeflow Community plans to deliver its v1.9 release in Jul 2024 per this [timeline](https://github.com/kubeflow/community/blob/master/releases/release-1.9/README.md#timeline). The high level deliverables are tracked in the [v1.9 Release](https://github.com/orgs/kubeflow/projects/61) Github project board. The v1.9 release process will be managed by the v1.9 [release team](https://github.com/kubeflow/community/blob/master/releases/release-1.9/release-team.md) using the best practices in the [Release Handbook](https://github.com/kubeflow/community/blob/master/releases/handbook.md).
+
+### Themes
+* Kubernetes 1.29 support
+* CNCF Graduation
```
Comment (zijianjoy) [inline ROADMAP.md:8]: We should aim for CNCF transition finished first before this Graduation effort.

File: ROADMAP.md [hunk 3]
```diff
@@ -1,6 +1,26 @@
...
+* LLM APIs
+* New component: Model Registry
+* Kubeflow Pipelines and kfp-tekton merged in a single GitHub repository
```
Comment (zijianjoy) [inline ROADMAP.md:11]: This is not what we have planned and it should be discussed. Let's remove this for now.

Comment (zijianjoy) [reply ROADMAP.md:11]: cc @chensun @terrytangyuan

Comment (terrytangyuan) [reply ROADMAP.md:11]: cc @rimolive @Tomcli

Comment (rimolive) [reply ROADMAP.md:11]: According to our last discussion, I'll resolve this one as we made an agreement of the ktp-tekton merge

File: ROADMAP.md (hunk 2 above)
Comment (rimolive) [reply ROADMAP.md:8]: Let me rephrase to "CNCF Transition" instead.

File: ROADMAP.md (hunk 3 above)
Comment (Tomcli) [reply ROADMAP.md:11]: Maybe add V2 to the release statement because we are still maintaining kfp-tekton v1 in a separate repo. Just to let others know kfp-tekton is still in active maintenance mode.

File: ROADMAP.md [hunk 4]
```diff
@@ -1,6 +1,26 @@
...
-## Kubeflow 1.8 Release, Planned for release: Oct 2023
+## Kubeflow 1.9 Release, Planned for release: Jul 2024
+The Kubeflow Community plans to deliver its v1.9 release in Jul 2024 per this [timeline](https://github.com/kubeflow/community/blob/master/releases/release-1.9/README.md#timeline). The high level deliverables are tracked in the [v1.9 Release](https://github.com/orgs/kubeflow/projects/61) Github project board. The v1.9 release process will be managed by the v1.9 [release team](https://github.com/kubeflow/community/blob/master/releases/release-1.9/release-team.md) using the best practices in the [Release Handbook](https://github.com/kubeflow/community/blob/master/releases/handbook.md).
+
+### Themes
+* Kubernetes 1.29 support
+* CNCF Transition
+* LLM APIs
+* New component: Model Registry
+* Kubeflow Pipelines and kfp-tekton merged in a single GitHub repository
```
Comment (Tomcli) [inline ROADMAP.md:11]:
```suggestion
* Kubeflow Pipelines and kfp-tekton V2 merged in a single GitHub repository
```

**Comment by thesuperzapper** [comment]
@rimolive please do NOT ever merge PRs using github, as it bypasses all the protections.

@james-jwu what is the current branch protection on the repo, and why was it possible for @rimolive to merge to master using the GitHub interface?
